cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -209,3 +215,53 @@ def remove_failed_flow_cell(self):
         self.remove_from_demultiplexed_runs()
         if self.files_exist_in_housekeeper and self.is_correctly_named:
             self.remove_files_from_housekeeper()
+
+    def archive_sample_sheet(self):
+        """Archives a sample sheet to /home/proj/production/sample_sheets and adds it to
+        Housekeeper with an appropriate tag"""
+        LOG.info("Archiving sample sheet for flow cell %s", self.run_name)
+
+        original_sample_sheet: Path = self.path / "SampleSheet.csv"
```

Comment:
```suggestion
        original_sample_sheet: Path = Path(self.path, "SampleSheet.csv")
```
I think we should make a constant of: SAMPLE_SHEET_FILE_NAME = "SampleSheet.csv", since you are using it more than once. Is this name always the same independent of sequencer used?